src/coreclr/runtime.proj

```diff
@@ -51,6 +51,7 @@
       <_CoreClrBuildArg Include="-cmakeargs &quot;-DCLR_DOTNET_HOST_PATH=$(DOTNET_HOST_PATH)&quot;" />
       <_CoreClrBuildArg Condition="'$(HasCdacBuildTool)' == 'true'" Include="-cmakeargs &quot;-DCDAC_BUILD_TOOL_BINARY_PATH=$(RuntimeBinDir)cdac-build-tool\cdac-build-tool.dll&quot;" />
       <_CoreClrBuildArg Condition="'$(_IcuDir)' != ''" Include="-cmakeargs &quot;-DCLR_CMAKE_ICU_DIR=$(_IcuDir)&quot;" />
+      <_CoreClrBuildArg Condition="'$(FeatureXplatEventSource)' == 'false'" Include="-cmakeargs &quot;-DCLR_CMAKE_DISABLE_XPLAT_EVENTSOURCE=1&quot;" />
```

Comment:
Can we reuse `FEATURE_EVENTSOURCE_XPLAT` instead of introducing new `CLR_CMAKE_DISABLE_XPLAT_EVENTSOURCE`?

Reply:
I assume we want `clrfeatures.cmake` to default to the current behavior, so here we'd like to say: disable the feature. That is what `CLR_CMAKE_DISABLE_XPLAT_EVENTSOURCE` is for. Does that make sense?

Reply:
It works!